src/main/java/com/plaid/client/response/Institution.java

```diff
@@ -221,4 +221,8 @@ public String getName() {
   public boolean getOAuth() {
     return oauth;
   }
+
+  public List<String> getRoutingNumbers() {
+    return routingNumbers;
```

Comment:
this file is the only real change in this PR that is not removing final from classes